sqlx-postgres/src/type_checking.rs

```diff
@@ -186,7 +192,7 @@ impl_database_ext! {
             Vec<sqlx::postgres::types::PgRange<sqlx::types::chrono::DateTime<_>>>,
 
         #[cfg(feature = "chrono")]
-        &[sqlx::postgres::types::PgRange<sqlx::types::chrono::DateTime<sqlx::types::chrono::Utc>>] |
+        Vec<sqlx::postgres::types::PgRange<sqlx::types::chrono::DateTime<sqlx::types::chrono::Utc>>> |
```

Comment:
This is theoretically a breaking change but this only affects code that wouldn't have compiled anyways as `&[PgRange<DateTime<Utc>>]`  doesn't implement `Decode`.

Reply:
Yeah, so because this change requires a type to actually implement `Encode` or `Decode` I can't really get around this.